apps/dialtone-documentation/docs/about/whats-new/posts/2024-4-11.md

```diff
@@ -0,0 +1,282 @@
+dist/lib/rich-text-editor.cjs                     14.56 kB │ gzip: 5.17 kB │ map: 52.99 kB
+dist/dialtone-vue.cjs                             14.98 kB │ gzip: 3.94 kB │ map:  0.10 kB
+dist/lib/editor.cjs                               14.98 kB │ gzip: 4.15 kB │ map: 28.14 kB
+dist/lib/popover.cjs                              14.98 kB │ gzip: 4.58 kB │ map: 41.03 kB
+dist/lib/emoji-picker.cjs                         23.10 kB │ gzip: 6.21 kB │ map: 50.83 kB
+✓ built in 2.15s
+```
+
+## Migration to dialtone >=9.27.1
+
+Replace any instances of the old imports with the corresponding new imports.
+Refer to the examples table below for clarity.
+
+| Previous 👎                                                                      | New 👍                                                                          | Enable Tree shaking (best) ✨                                                        |
+|----------------------------------------------------------------------------------|---------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|
+| import { DtEmoji } from '@dialpad/dialtone/vue/emoji'                            | import { DtEmoji } from '@dialpad/dialtone/vue'                                 | import { DtEmoji } from '@dialpad/dialtone/vue/lib/emoji.js'                        |
+| import { DtMessageInput } from '@dialpad/dialtone/vue/message_input'             | import { DtMessageInput } from '@dialpad/dialtone/vue'                          | import { DtMessageInput } from '@dialpad/dialtone/vue/lib/message-input.js'         |
+| import { DtTooltipDirective } from '@dialpad/dialtone/vue/directives'            | import { DtTooltipDirective } from '@dialpad/dialtone/vue'                      | import { DtTooltipDirective } from '@dialpad/dialtone/vue/lib/tooltip-directive.js' |
+| import SpotBlankSpace from '@dialpad/dialtone/dist/css/vue/spot/SpotBlankSpace'; | import SpotBlankSpace from '@dialpad/dialtone/css/vue/spot/SpotBlankSpace.vue'; | -                                                                                   |
+
+## Using dialtone-icons
+
+Backwards-compatible support for DtIcon will remain for the near-term. However, we encourage you to begin updating your projects to the new dialtone-icons usage for a seamless and better experience.
+
+Previously, to use an icon on your app you had to do:
+
+```jsx
+import { DtIcon } from '@dialpad/dialtone'
+
+<dt-icon name="accessibility" size="300" />
+```
+
+This is going to change going forward to enable better tree shaking capabilities and a possible lazy-loading in the future to improve even more the load of your app.
+
+From now on, if you need an icon, you can use it as following:
+
+```jsx
+import { DtIconAccessibility } from '@dialpad/dialtone-icons'
+
+<dt-icon-accessibility size="300" />
+```
```

Comment:
Is it the same difference as the components where it can also be imported like `import { DtIconAccessibility } from '@dialpad/dialtone-icons/accessibility'`?

Reply:
Yes, it's exactly the same, I'll update to add both imports